### ISSUE TITLE
Update blew up

### ISSUE THREAD

**Issue by DomT4** [robbyrussell/oh-my-zsh#3576]
Doing my standard automatic 3 day update cycle of ` Oh-my-zsh ` it didn't go smoothy. As far as I can see, I didn't change anything to cause the failure, I haven't modified the file involved at all.

```
 34 files changed, 273 insertions(+), 248 deletions(-)
 delete mode 100644 lib/aliases.zsh
First, rewinding head to replay your work on top of it...
Applying: Brings all directory stuff to directories.zsh
Using index info to reconstruct a base tree...
A	lib/aliases.zsh
M	lib/directories.zsh
Falling back to patching base and 3-way merge...
Auto-merging lib/directories.zsh
CONFLICT (content): Merge conflict in lib/directories.zsh
CONFLICT (modify/delete): lib/aliases.zsh deleted in HEAD and modified in Brings all directory stuff to directories.zsh. Version Brings all directory stuff to directories.zsh of lib/aliases.zsh left in tree.
Failed to merge in the changes.
Patch failed at 0001 Brings all directory stuff to directories.zsh
The copy of the patch that failed is found in:
   /Users/Dominyk/.oh-my-zsh/.git/rebase-apply/patch

When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort".

There was an error updating. Try again later?
/Users/Dominyk/.oh-my-zsh/lib/directories.zsh:25: parse error near `<<<'
```

I just ` cd `-ed into the directory and did a hard git reset and then a ` git pull origin master ` which solved the issue. Just thought I'd flag it up in case it's anything that needs fixing/changing upstream here.

**Comment by mcornella** [comment]
It seems you had some modification in the oh-my-zsh folder that conflicted with the rebase. I tried updating yesterday with a clean repository and no conflicts appeared. 

To be sure, you could post the output of the recent reflog with `git reflog --relative-date` and we'll look at the changes you had.

**Comment by DomT4** [comment]
```
ef7e53a HEAD@{18 hours ago}: reset: moving to origin/master
e6b1319 HEAD@{18 hours ago}: rebase: aborting
ef7e53a HEAD@{18 hours ago}: pull --rebase --stat origin master: checkout ef7e53a78d0e4196c2d1e6e5b268209759d51753
e6b1319 HEAD@{13 days ago}: rebase finished: returning to refs/heads/master
e6b1319 HEAD@{13 days ago}: pull --rebase --stat origin master: Fix a typo
a705c2f HEAD@{13 days ago}: pull --rebase --stat origin master: Deletes dot aliases from common-aliases.
87e69ea HEAD@{13 days ago}: pull --rebase --stat origin master: Moves a bindkey call from misc to key-bindings.
5c65396 HEAD@{13 days ago}: pull --rebase --stat origin master: Moves remaining aliases to misc and deletes file.
5eff79b HEAD@{13 days ago}: pull --rebase --stat origin master: Refactors history alias and moves it.
7f0534d HEAD@{13 days ago}: pull --rebase --stat origin master: Pushes autoenv code to its plugin.
5a4d8e6 HEAD@{13 days ago}: pull --rebase --stat origin master: Brings all directory stuff to directories.zsh
550ccca HEAD@{13 days ago}: pull --rebase --stat origin master: checkout 550ccca91cb45fcf26a8f2c3c60da675b995e3f3
7344977 HEAD@{4 weeks ago}: rebase finished: returning to refs/heads/master
7344977 HEAD@{4 weeks ago}: pull --rebase --stat origin master: Fix a typo
c6f0004 HEAD@{4 weeks ago}: pull --rebase --stat origin master: Deletes dot aliases from common-aliases.
f6e9d40 HEAD@{4 weeks ago}: pull --rebase --stat origin master: Moves a bindkey call from misc to key-bindings.
fb2ca88 HEAD@{4 weeks ago}: pull --rebase --stat origin master: Moves remaining aliases to misc and deletes file.
1ab885f HEAD@{4 weeks ago}: pull --rebase --stat origin master: Refactors history alias and moves it.
5c6dbd8 HEAD@{4 weeks ago}: pull --rebase --stat origin master: Pushes autoenv code to its plugin.
c238d01 HEAD@{4 weeks ago}: pull --rebase --stat origin master: Brings all directory stuff to directories.zsh
c78277f HEAD@{4 weeks ago}: pull --rebase --stat origin master: checkout c78277fd8bda5fec87504469afdf121355876006
9107eb0 HEAD@{8 weeks ago}: rebase finished: returning to refs/heads/master
```

**Comment by mcornella** [comment]
It looks like you merged at some point PR #2422, which was merged into the project this last update. So there's your rebase conflict. You're safe now :smile: 
Don't forget to close this issue!

**Comment by DomT4** [comment]
Oh! Yes, I think I did. I wildly forgot about that, and now feel a little silly :wink:. Thanks!
